README.md

```diff
@@ -74,7 +63,7 @@ Valid `opts` include:
 - `opts.encryptionKey` (string): optional encryption key to use during replication. If not provided, a default insecure key will be used.
 - `opts.hypercore`: constructor of a hypercore implementation. `hypercore@8.x.x` is used from npm if not provided.
 
-### multi.writer([name, ]cb)
+### multi.writer([name], [keypair], cb)
```

Comment:
Could we use `opts` instead of `keypair`? This'll make it easier to further expand this API in the future without adding further params.

Reply:
So maybe it'd be `opts.keypair.{secret,public}`